### PR TITLE
ARM pseudo subvar: don't substitude [pc, reg]

### PR DESCRIPTION
**Pull request by pluswave**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

if code disassam is like 
```` 
ldr r0, [pc, r0] 
````
show as it , don't subsitude address.

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Comment by XVilka** [comment]
Please fix the tests: https://github.com/rizinorg/rizin/runs/7427772183?check_suite_focus=true#step:12:751
Looks like new output is better

**Review by XVilka — CHANGES_REQUESTED** [review]
Please use the `clang-format`, until linter CI job is green

File: librz/parse/p/parse_arm_pseudo.c [hunk 1]
```diff
@@ -290,16 +290,20 @@ static bool subvar(RzParse *p, RzAnalysisFunction *f, RzAnalysisOp *op, char *da
 			if (!ripend) {
 				ripend = "]";
 			}
-			if (neg) {
-				repl_num -= rz_num_get(NULL, neg + 1);
-			} else {
-				repl_num += rz_num_get(NULL, rip);
+			const char * maybe_num = neg ? neg + 1 : rip;
+			while (*maybe_num == ' ') maybe_num ++;
```
Comment (XVilka) [inline librz/parse/p/parse_arm_pseudo.c]: `rz_str_trim_head_ro()` could be used instead probably

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/parse/p/parse_arm_pseudo.c (hunk 1 above)
Comment (pluswave) [reply librz/parse/p/parse_arm_pseudo.c]: changed already.

**Review by wargio — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Pull request merged**